pkg/arvo/ted/graph/join.hoon

```diff
@@ -0,0 +1,74 @@
+/-  spider, graph-view, graph=graph-store, *metadata-store, *group
+/+  strandio, resource
+=>
+|% 
+++  strand  strand:spider
+++  fail    strand-fail:strand
+++  poke  poke:strandio
+++  poke-our   poke-our:strandio
+::
+++  check-live
```

Comment:
Why does this have a timeout loop?  I think this should be handled by Ames, not here.